framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -313,6 +313,10 @@
     // org.checkerframework.framework.util.typeinference.DefaultTypeArgumentInference
     "showInferenceSteps",
 
+    // Output stack trace when checker encounters errors
+    // org.checkerframework.common.basetype.BaseTypeChecker.printStackTrace()
+    "dumpOnErrors",
```

Comment:
The `printOrStoreMessage` in this class should also handle this flag. So you might need to move the `printStackTrace` into this class and make it protected.